include/fast_float/constexpr_feature_detect.h

```diff
@@ -14,6 +14,12 @@
 #define FASTFLOAT_CONSTEXPR14
 #endif
 
+#if __cplusplus >= 201703L
```

Comment:
MSVC doesn't always define `__cplusplus` correctly, see below. Use the following instead, or introduce a `FASTFLOAT_CPLUSPLUS_VERSION` macro:

```c
#if __cplusplus >= 201703L || (defined(_MSVC_LANG) && _MSVC_LANG >= 201703L)
```